src/main/kotlin/com/manimdsl/runtime/ASTExecutor.kt

```diff
@@ -39,7 +39,12 @@ class VirtualMachine(
             if (statements[it + 1] !is NoRenderAnimationNode &&
                 (acceptableNonStatements.any { x -> fileLines[it].contains(x) } || statements[it + 1] is CodeNode)
             ) {
-                displayCode.add(fileLines[it])
+                if (fileLines[it].isEmpty()){
+                    displayCode.add(" ")
```

Comment:
What does adding a space do?

Reply:
The code scrolling gets messed up within the Manim `Code`object when empty strings are passed in

Reply:
Sure